.github/workflows/e2eRemote.yml

```diff
@@ -32,21 +32,11 @@ on:
 
 jobs:
   
-  cypress-remote-test:
+  cypress_remote_test:
```

Comment:
do we actually use this 🤔

Reply:
We don't at the moment! I have every intention of enabling this job in the coming months. :smile:

Reply:
Oh!! Got it thank you for the clarification!! 🎉